src/containers/TransferCurrency/TransferPage.tsx

```diff
@@ -191,14 +191,17 @@ export class TransferPagePresentation extends React.Component<Props, State> {
               <section className="columns">
                 <div className="column content has-text-grey-lighter">
                   <p>Transaction fee</p>
+                  <p>Network fee</p>
                   <p>Remaining balance</p>
                 </div>
                 <div
                   className={`column has-text-right content ${addMetalColour(this.props.currency)}`}
                 >
                   <p>
-                    {Number(this.props.formData.fee).toFixed(5) || 0} {this.props.currency}
+                    {(Number(this.props.formData.fee) - 100e-7).toFixed(5) || 0}{' '}
```

Comment:
I'd prefer to store the value of this specific number `100e-7` as a constant or something, as it's used in a few places. It would make more sense when reading and will be easy to manage if it ever changes etc. Is it the base fee ?